### PR TITLE
port toranj test40 network_data_stable_full

### PR DESCRIPTION
**Pull request by priyankaCh22**

### CODE REVIEW

File: silk/tests/openthread/ot_test_network_data_stable_full.py [hunk 1]
```diff
@@ -0,0 +1,313 @@
...
+
+        self.logger.info(f"========= Remove temporary prefix {prefix2} from leader node =========")
+        self.leader.remove_prefix(prefix2)
+        self.prefix_add_remove()
+
+        self.logger.info(f"-------- Add off-mesh route based on temporary prefix {prefix1} on leader node --------")
+        self.leader.add_route_using_prefix(prefix1, stable=False)
+        # Wait till network data is updated and all nodes see all the added off-mesh routes.
+        time.sleep(WAIT_TIME)
+        num_routes = num_routes + 1
```
Comment (qilinkang) [inline silk/tests/openthread/ot_test_network_data_stable_full.py]: num_routes += 1

File: silk/tests/openthread/ot_test_network_data_stable_full.py [hunk 2]
```diff
@@ -0,0 +1,313 @@
...
+        self.prefix_add_remove()
+        verify(len(wpan_table_parser.parse_list(self.ed2.get(wpan.WPAN_THREAD_OFF_MESH_ROUTES))) == num_routes)
+
+        self.logger.info(f"-------- Add off-mesh route based on stable prefix {prefix2} on leader node --------")
+        self.leader.add_route_using_prefix(prefix2, stable=True)
+        # Wait till network data is updated and all nodes see all the added off-mesh routes.
+        time.sleep(WAIT_TIME)
+        num_routes = num_routes + 1
```
Comment (qilinkang) [inline silk/tests/openthread/ot_test_network_data_stable_full.py]: num_routes += 1

File: silk/tests/openthread/ot_test_network_data_stable_full.py [hunk 3]
```diff
@@ -0,0 +1,313 @@
...
+        self.leader.add_prefix(prefix3, stable=True)
+        # Wait till network data is updated and all nodes see all the added off-mesh routes.
+        self.prefix_add_remove()
+        verify(len(wpan_table_parser.parse_list(self.ed2.get(wpan.WPAN_THREAD_OFF_MESH_ROUTES))) == num_routes)
+
+        self.logger.info(f"-------- Remove off-mesh route based on stable prefix {prefix2} on leader node --------")
+        self.leader.remove_route(prefix2)
+        # Wait till network data is updated and all nodes see all the added off-mesh routes.
+        time.sleep(WAIT_TIME)
+        num_routes = num_routes - 1
```
Comment (qilinkang) [inline silk/tests/openthread/ot_test_network_data_stable_full.py]: num_routes -= 1

File: silk/tests/openthread/ot_test_network_data_stable_full.py [hunk 4]
```diff
@@ -0,0 +1,313 @@
...
+        self.prefix_add_remove()
+        verify(len(wpan_table_parser.parse_list(self.ed2.get(wpan.WPAN_THREAD_OFF_MESH_ROUTES))) == num_routes)
+
+        self.logger.info(f"-------- Remove off-mesh route based on temporary prefix {prefix1} on leader node --------")
+        self.leader.remove_route(prefix1)
+        # Wait till network data is updated and all nodes see all the added off-mesh routes.
+        time.sleep(WAIT_TIME)
+        num_routes = num_routes - 1
```
Comment (qilinkang) [inline silk/tests/openthread/ot_test_network_data_stable_full.py]: num_routes -= 1

**Review by qilinkang — COMMENTED** [review]
How is the test run ?

File: silk/tests/openthread/ot_test_network_data_stable_full.py (hunk 1 above)
Comment (priyankaCh22) [reply silk/tests/openthread/ot_test_network_data_stable_full.py]: Ack

File: silk/tests/openthread/ot_test_network_data_stable_full.py (hunk 2 above)
Comment (priyankaCh22) [reply silk/tests/openthread/ot_test_network_data_stable_full.py]: Ack

File: silk/tests/openthread/ot_test_network_data_stable_full.py (hunk 3 above)
Comment (priyankaCh22) [reply silk/tests/openthread/ot_test_network_data_stable_full.py]: Ack

File: silk/tests/openthread/ot_test_network_data_stable_full.py (hunk 4 above)
Comment (priyankaCh22) [reply silk/tests/openthread/ot_test_network_data_stable_full.py]: Ack

**Comment by priyankaCh22** [comment]
> How is the test run ?

Test run is good all subtests are able to PASS.
[2020-10-15 09:42:45,206] [silk] [INFO] ============================= SUMMARY ===============================
[2020-10-15 09:42:45,206] [silk] [INFO] ======================================================================
[2020-10-15 09:42:45,206] [silk] [INFO] TestNetworkDataStableFull
[2020-10-15 09:42:45,206] [silk] [INFO]     test01_clear_full_network_data_on_sed1......................PASS
[2020-10-15 09:42:45,207] [silk] [INFO]     test02_pairing..............................................PASS
[2020-10-15 09:42:45,207] [silk] [INFO]     test03_verify_network_data_on_prefix_add_remove.............PASS
[2020-10-15 09:42:45,207] [silk] [INFO]     test04_verify_add_remove_same_prefix........................PASS
[2020-10-15 09:42:45,207] [silk] [INFO] ======================================================================

**Review by qilinkang — APPROVED** [review]
LGTM

**Review by jwhui — APPROVED** [review]

**Pull request merged**